lib/BuildSystem/BuildSystem.cpp

```diff
@@ -2853,12 +2853,30 @@ class SwiftCompilerShellCommand : public ExternalCommand {
     return ExternalCommand::configureAttribute(ctx, name, values);
   }
 
+  template <typename String>
+  std::string escapeForJson(String const& in) const {
+    std::ostringstream ss;
+    for (auto i = in.begin(); i != in.end(); i++) {
+      switch (*i) {
+      case '\\': ss << "\\\\"; break;
+      case '"': ss << "\\\""; break;
+      case '/': ss << "\\/"; break;
+      case '\b': ss << "\\b"; break;
+      case '\f': ss << "\\f"; break;
```

Comment:
json.org doesn't indicate that `\f` should be escaped.

Reply:
Yes it does.